Ryujinx.Ava/UI/Windows/MainWindow.axaml.cs

```diff
@@ -25,63 +21,43 @@
 using Ryujinx.Ui.Common;
 using Ryujinx.Ui.Common.Configuration;
 using Ryujinx.Ui.Common.Helper;
-using SixLabors.ImageSharp.PixelFormats;
 using System;
 using System.ComponentModel;
 using System.IO;
-using System.Threading;
 using System.Threading.Tasks;
 using InputManager = Ryujinx.Input.HLE.InputManager;
 
 namespace Ryujinx.Ava.UI.Windows
 {
     public partial class MainWindow : StyleableWindow
-    {
+    {        
```

Comment:
```suggestion
    {
```
I recommend configuring your IDE to remove trailing whitespace automatically if you can.